packages/gatsby/cache-dir/default-html.js

```diff
@@ -10,7 +10,7 @@ export default class HTML extends React.Component {
           <meta httpEquiv="x-ua-compatible" content="ie=edge" />
           <meta
             name="viewport"
-            content="width=device-width, initial-scale=1, shrink-to-fit=no"
+            content="width=device-width, initial-scale=1, shrink-to-fit=no, viewport-fit=cover"
```

Comment:
I'm not sure we want to make this the default. This would presume that _every_ website using Gatsby will make the necessary changes for iPhone X compatibility, correct?

Reply:
Maybe pass this in with react-helmet instead? I'd think the _last_ one will win out, so that should be a good solution here.

Reply:
Correct, this will affect every website using Gatsby! That would be a bit overkill. 
I added it to the react-helmet tags in `layout.js`.